### DISCUSSION TITLE
Dialog doesn't allow interaction with 1password browser extension

### DISCUSSION THREAD

**Discussion by ajosedev** [ariakit/ariakit#862] [Questions] [answered]
Hi there 👋 . Just started using `reakit`, and having a very specific issue when creating an authentication form inside of a `Dialog` component. I totally understand if this isn't fixable on your end and is instead specific to this one extension. As 1password seems to be using Web Components, this _may_ be more broadly applicable.

When focusing on an input field inside of a `Dialog`, interaction to the 1password tooltip is seemingly blocked. I've tried to debug with not much success, but my best guess is that the `useFocusOnBlur` function doesn't see the tooltip as an actual element (`isActualElement`), so focus is returned to the dialog instead.

https://user-images.githubusercontent.com/29142528/116018378-46958f80-a685-11eb-9f16-1f2084a21efc.mp4

Some other things I've tried that may be helpful:
- Passing `modal={false}` to the `Dialog` has no effect. This allows focus to escape the modal, but focus is still not passed to the tooltip
- Passing `hideOnClickOutside={false} also does nothing
- Removing `tabindex` from the DOM of the `role=dialog` fixes the problem. Changing the `tabindex` to any other number seems to do nothing.

I've created a minimal repro here on CodeSandbox: https://codesandbox.io/s/festive-euler-m49wt?file=/src/App.js.

Thankyou!

**Comment by ZachHaber** [comment]
I can confirm that I can reproduce this. Chrome with [1Pasword extension](https://chrome.google.com/webstore/detail/1password-%E2%80%93-password-mana/aeblfdkhhhdcdjpifhhbdiojplfjncoa)

**Comment by diegohaz** [comment, marked as answer]
Good catch @ajosedev! Thanks! Could you please create an issue so we can better track it?

**Reply by ajosedev** [reply]
Thanks @diegohaz, maybe should have put it there in the first place 😅. I've created an issue here: https://github.com/reakit/reakit/issues/863
